<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeConnectingSettings</class>
 <widget class="QWidget" name="ChangeConnectingSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>273</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>390</width>
    <height>273</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>390</width>
    <height>273</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="hostLbl">
     <property name="text">
      <string>host</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="hostEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="userNameLbl">
     <property name="text">
      <string>username</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="userNameEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>password</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QLineEdit" name="passwordEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="databasenameLabel">
     <property name="text">
      <string>databasename</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1" colspan="2">
    <widget class="QLineEdit" name="databasenameEdit"/>
   </item>
   <item row="4" column="1">
    <widget class="QPushButton" name="btnCancel">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>131</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Отмена</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QPushButton" name="btnApply">
     <property name="minimumSize">
      <size>
       <width>131</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>Применить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
